sandbox/apparmor/notify/listener/listener.go

```diff
@@ -410,7 +418,12 @@ func (l *Listener) waitAndRespondAaClassFile(req *Request, msg *notify.MsgNotifi
 	var allow bool
 	select {
 	case reply := <-req.replyChan:
-		allow = reply.(bool)
+		var ok bool
+		allow, ok = reply.(bool)
+		if !ok {
+			logger.Debugf("invalid reply from client: %v; denying request", reply)
+			allow = false
+		}
```

Comment:
now, we can also do a check in the Request.Reply as well, no?

Reply:
We could, yes. I was thinking it would be better to send back a deny immediately rather than expect the client to re-reply with a well-formed response. But you're right, if we error here, we can presumably pass that error back to the caller, and out over the API to the client as well.

Reply:
I added a check and test for this.